docker-compose.yaml

```
@@ -0,0 +1,112 @@
+name: opik
+
+services:
+  mysql:
+    image: mysql:8.4.2
+    hostname: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: opik
+      MYSQL_DATABASE: opik
+      MYSQL_USER: opik
+      MYSQL_PASSWORD: opik
+    healthcheck:
+      test: ["CMD", "mysqladmin", "ping", "-h", "127.0.0.1", "--silent"]
+      timeout: 1s
+      interval: 1s
+      retries: 300
+    ports:
+      - "3306:3306"
+
+  redis:
+    image: redis:7.2.4-alpine3.19
+    hostname: redis
+    command: redis-server --requirepass opik
+    ports:
+      - '6379:6379'
+    healthcheck:
+      test: [ "CMD", "nc", "-z", "localhost", "6379" ]
+      interval: 2s
+      timeout: 4s
+      retries: 20
+      start_period: 30s
+    restart: always
+
+
+  clickhouse:
+    image: clickhouse/clickhouse-server:23.8.15.35-alpine
+    hostname: clickhouse
+    environment:
+      CLICKHOUSE_DB: opik
+      CLICKHOUSE_USER: opik
+      CLICKHOUSE_PASSWORD: opik
+      # Enables SQL-driven Access Control and Account Management:
+      # https://clickhouse.com/docs/en/operations/access-rights#enabling-access-control
+      CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT: 1
+    ports:
+      - "8123:8123" # HTTP default port
+      - "9000:9000" # Native Protocol port
+    volumes: # Mounted on your $HOME folder
+      - ~/clickhouse/data:/var/lib/clickhouse/
+      - ~/clickhouse/logs:/var/log/clickhouse-server/
+    healthcheck:
+      test: [ "CMD", "wget", "--spider", "-q", "http://127.0.0.1:8123/ping" ]
+      interval: 1s
+      timeout: 1s
+      retries: 300
+
+  backend:
+    image: opik-backend
```

```suggestion
    image: ghcr.io/comet-ml/opik/opik-backend:${OPIK_VERSION:-latest}
```

Allows users to download the latest image or a specific version if they don't want to build the images

---

see 2 compose files and the readme